src/main.js

```diff
@@ -60,34 +60,66 @@ const router = new VueRouter({
         join('/')+'/'
 });
 
-// Route Guard rules for directing users
-router.beforeEach((to, from, next) => {
-    var auth = Store.netMgr.isAuth();
+// Nasty hack to get round the new NavigationErrors
+const originalPush = VueRouter.prototype.push
+VueRouter.prototype.push = function push(
```

Comment:
aren't we duplicating `VueRouter.prototype.push`?

Reply:
😬 
yes, so, Vue-Router 3.0 has decided to throw an exception if you
- have a page that programmatically redirects
- navigates back to the page we're on

The official solution is to 
```
.catch(function() { ... deal with errors here ... })
``` after every push...
which this function effectivly tries to do.

It's not a long term solution, but it does get us running